packages/form-js-editor/src/features/properties-panel/groups/GeneralGroup.js

```diff
@@ -41,6 +41,10 @@ export default function GeneralGroup(field, editField, getService) {
     ...ReadonlyEntry({ field, editField })
   ];
 
+  if (entries.length === 0) {
```

Comment:
Why is this necessary?

Reply:
So that the group doesn't show up when no entries are within in, the separator currently doesn't have any configuration. It's standard pattern we just never had a field without general entries before.